src/main/java/org/stellar/sdk/requests/TradesRequestBuilder.java

```diff
@@ -103,8 +104,11 @@ public TradesRequestBuilder forTradeType(String tradeType) {
         return this.execute(this.httpClient, this.buildUri());
     }
 
-    public TradesRequestBuilder offerId(String offerId) {
-        uriBuilder.setQueryParameter("offer_id", offerId);
+    public TradesRequestBuilder offerId(Long offerId) {
+        if (offerId == null) {
+            return this;
```

Comment:
maybe this should unset the query parameter?

Reply:
good point on builder, included that, added unit test coverage to assert, thanks! [33ea8dce2deb](https://github.com/stellar/java-stellar-sdk/pull/386/commits/33ea8dce2deb284a2d7f4b0958cda067f88bc269)